src/index.html

```diff
@@ -3,11 +3,297 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap" rel="stylesheet">
+    <title>B&O</title>
+    <link rel="stylesheet" href="./main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <a class="logo" href="#">
+          <img class="logo__icon"
+          src="images/icons/Logo.svg" alt="logo">
+        </a>
+
+        <div class="navigation__buttons">
+
+          <a class="phone" href="tel:+1 234 555-55-55">
+            <span class="phone__label">tel:+1 234 555-55-55</span>
+            <img
+              class="phone__icon"
+              src="images/icons/Icon-Phone-call.svg"
+              alt="icon-phone">
```

Comment:
Fix formatting. Also alt attribute is used by screen readers so it should be like:
```suggestion
            <img
              class="phone__icon"
              src="images/icons/Icon-Phone-call.svg"
              alt="phone icon"
            >
```